### PR TITLE
Update RUN_WITH_EXTENSION.md

### PR DESCRIPTION
**Pull request by MingjiaLiu1995**
Add a section to guide developers to explore sample codes per developer's feedback.

### CODE REVIEW

File: Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md [hunk 1]
```diff
@@ -29,10 +29,16 @@ The simpliest way to run this add-in project is using Office Add-ins Development
     If you still have problems, check [troubleshoot development errors]( https://learn.microsoft.com/en-us/office/dev/add-ins/testing/troubleshoot-development-errors) or [Create an issue](https://aka.ms/officedevkitnewissue) and we'll help you out.  
 
 
+# <br>Explore the add-in code
+To explore an Office add-in project, you can start with the key files listed below:
+
+- The `./manifest.xml` file in the root directory of the project defines the settings and capabilities of the add-in.  <br>You can check whether your manifest file is valid by selecting `Validate Manifest File` in the `Office Add-ins Development Kit` extension tree view.
```
Comment (AlexJerabek) [inline Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md:35]: I don't think the term "tree view" is a well understood UI construct. I'd recommend avoiding it and just referring to options in the extension. Also, UI buttons should be **bold**, not `code`.
```suggestion
- The `./manifest.xml` file in the root directory of the project defines the settings and capabilities of the add-in.  <br>You can check whether your manifest file is valid by selecting **Validate Manifest File** option from the Office Add-ins Development Kit.
```

File: Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md [hunk 2]
```diff
@@ -29,10 +29,16 @@ The simpliest way to run this add-in project is using Office Add-ins Development
...
+- The `./src/taskpane/taskpane.html` file contains the HTML markup for the task pane.
+- The `./src/taskpane/taskpane.css` file contains the CSS that's applied to content in the task pane.
+- The `./src/taskpane/taskpane.js` file contains the Office JavaScript API code that facilitates interaction between the task pane and the Excel application.
+
 # See also
-    
-* **See detailed introduction to this add-in project:** `README.md` file.
-* **Explore More add-in samples:** `View Samples` in `Office Add-ins Development Kit` tree view.
+* **Explore more add-in samples:** `View Samples` in `Office Add-ins Development Kit` tree view.
```
Comment (AlexJerabek) [inline Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md:41]:
```suggestion
* **Explore more add-in samples:** Select **View Samples** in the Office Add-ins Development Kit.
```

File: Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md [hunk 3]
```diff
@@ -29,10 +29,16 @@ The simpliest way to run this add-in project is using Office Add-ins Development
...
 * **Read the documentation:** [Office add-ins documentation](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins)
 * **Experience any problems?** [Create an issue](https://aka.ms/officedevkitnewissue) and we'll help you out.
 * **Engage with the team to learn more about updates:** [Join the Microsoft Office Add-ins community call.](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins-community-call)
```
Comment (AlexJerabek) [inline Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md:44]: Slight style thing but avoid having periods in links.
```suggestion
* **Engage with the team to learn more about updates:** [Join the Microsoft Office Add-ins community call](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins-community-call).
```

File: Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md [hunk 4]
```diff
@@ -29,10 +29,16 @@ The simpliest way to run this add-in project is using Office Add-ins Development
     If you still have problems, check [troubleshoot development errors]( https://learn.microsoft.com/en-us/office/dev/add-ins/testing/troubleshoot-development-errors) or [Create an issue](https://aka.ms/officedevkitnewissue) and we'll help you out.  
 
 
+# <br>Explore the add-in code
+To explore an Office add-in project, you can start with the key files listed below:
```
Comment (AlexJerabek) [inline Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md:33]: There should only be one H1 per file. Also, if the sentence before a list is a complete sentence, it should end with a period, not a colon.
```suggestion
## Explore the add-in code
To explore an Office add-in project, you can start with the key files listed below.
```

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
There are also a few typos in line 14. Consider using a spell checker extension for VS Code (I use Code Spell Checker).

**Comment by MingjiaLiu1995** [comment]
Hi @AlexJerabek, I have updated the .md file according to your edits on the Excel Chart sample. Would you please help give it a final review? Then we will apply this RUN_WITH_EXTENSION.md to all samples. Thanks!

Btw, do you think the file name "RUN_WITH_EXTENSION" makes sense to you?

File: Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md [hunk 5]
```diff
@@ -1,38 +1,48 @@
 # Run the add-in with Office Add-ins Development Kit extension
-The simpliest way to run this add-in project is using Office Add-ins Development Kit. Here are steps to follow:
+We recommend you try this sample by using the [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger). The Office Add-ins Development Kit is an end-to end developer tool for building Office add-ins. It helps create, run, and debug an Office Add-in.
```
Comment (AlexJerabek) [inline Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md:2]:
```suggestion
We recommend you try this sample by using the [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger). The Office Add-ins Development Kit is an end-to end developer tool for building Office Add-ins. It helps create, run, and debug an Office Add-in.
```

**Review by AlexJerabek — APPROVED** [review]
Approving with one suggestion.

**Review by hermanwenhe — APPROVED** [review]

**Pull request merged**